### PR TITLE
refactor:implement pagination to view loan/saving/share in client page 

### PR DESCRIPTION
**Pull request by meetcric**
## Description
Implement pagination to view the loans/savings/Shares in the client page

## Related issues and discussion
#133 

## Screenshots, if any
screenshot from 2018-12-21 04-28-07
## Checklist
Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] If you have multiple commits please combine them into one commit by squashing them.

- [x] Read and understood the contribution guidelines at web-app/.github/CONTRIBUTING.md.

### CODE REVIEW

File: src/app/clients/view-client/view-client.component.ts [hunk 1]
```diff
@@ -158,7 +161,7 @@ export class ViewClientComponent implements OnInit, OnDestroy {
           this.clientDocuments = res;
           this.docId = res.id;
           this.dataSourceDocuments = new MatTableDataSource(res);
-          this.dataSourceDocuments.paginator = this.paginator;
+          // this.dataSourceDocuments.paginator = this.paginator;
```
[the remark this answers is not in the document]
Comment (meetcric) [reply src/app/clients/view-client/view-client.component.ts]: @abhaychawla Changes completed !! Sorry for the silly mistakes.

**Comment by meetcric** [comment]
@abhaychawla Changes completed !! Sorry for the silly mistakes.

File: src/app/clients/view-client/view-client.component.html [hunk 2]
```diff
@@ -118,6 +118,7 @@ <h3>{{clientInfo?.displayName}}</h3>
 
               </tr>
             </table>
+            <mat-paginator #loanPaginator [pageSizeOptions]="[10,25,50,100]" showFirstLastButtons></mat-paginator>
```
Comment (abhaychawla) [inline src/app/clients/view-client/view-client.component.html]: I asked you to ensure that there is a single space between `,` and page size option (Example: 25). Don't remove the space. Update this for all instances.
```suggestion
            <mat-paginator #loanPaginator [pageSizeOptions]="[10, 25, 50, 100]" showFirstLastButtons></mat-paginator>
```

**Comment by meetcric** [comment]
Thanks for giving your time to review the pull request @abhaychawla

**Pull request merged**